### DISCUSSION TITLE
Set of deformers

### DISCUSSION THREAD

**Discussion by tomDreams** [opentoonz/opentoonz#6758] [Ideas]
I truly Want a Set of deformers like may the i can just connect it whitout rig or plastic tool that ruining my vector lines 

deformer that squash and streach  that i can animate whit 
just click and you have it on your mesh 
you can add points in all directions 
![image](https://user-images.githubusercontent.com/85890154/121916781-2311cb80-cd3d-11eb-8051-2835615b714f.png)

**Comment by artisteacher** [comment]
The selection tool might already be close to what you are looking for - though it doesn't make keyframes. For just width/height - the selection box can be adjusted normally. For more complex deformation, after making a selection, hold down on Ctrl (or Cmd on mac) when dragging a corner - easy deformation.

**Comment by tomDreams** [comment]
I NEED THE KEYFRAMES THAS THE MAIN THING THAT I NEED


‫בתאריך יום ד׳, 16 ביוני 2021 ב-4:36 מאת ‪artisteacher‬‏ <‪
***@***.***‬‏>:‬
